lib/util/env/src/lib.rs

```diff
@@ -194,10 +195,20 @@ pub mod cloudflare {
 		}
 	}
 
+	pub fn account_id() -> &'static str {
```

Comment:
can't this be simplified to make the lazy_static store the result with EnvVarError instead of an option?